lighthouse-core/report/v2/report-styles.css

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ :root {
+  --text-font-family: '.SFNSDisplay-Regular', 'Helvetica Neue', 'Lucida Grande', sans-serif;
+  --body-font-size: 13px;
+
+  --secondary-text-color: #565656;
```

Comment:
> One thing that would also be interesting is offloading polyfills and broader browser support to Viewer. The more code we to ship in the binary, the larger the report output becomes.

Definitely, I see a world in which a hosted polyfill'd/browserify'd/whatever'd renderer bundle being included just via a single `<script src="jsdelivr.net/lighthouse/v2.x>` in WPT-style reports via CLI flag or something and DevTools/opt-in CLI users being the only ones to consume the source directly

> I'd be curious to see the analytics for WPT though.

I think this is the more important information to track when considering if we need to support Edge